scripts/tests/secrets-integration.sh

```diff
@@ -33,7 +33,7 @@ if NO_COLOR=0 "$earthly" account logout > output 2>&1; then
     echo "earthly account logout should have failed"
     exit 1
 fi
-diff output <(echo "Error: account logout has no effect when --auth-token (or the EARTHLY_TOKEN environment variable) is set")
+acbgrep "Error: account logout has no effect when --auth-token (or the EARTHLY_TOKEN environment variable) is set" output
```

Comment:
This should be changed back to the original test -- repeating all errors twice (in all cases) seems like a bad decision.

Reply:
I'll take another look. I asked Vlad about the original intention. He said it seems fine for now. :shrug: